modules/client/examples/05-encoders-decoders/05-register-standard-callback.ts

```diff
@@ -0,0 +1,65 @@
+/* MARKDOWN
+---
+title: Register Callback
+---
+
+## Register a New Standard Callback
+
+Register a new standard callback for the DAO.
+
+### Encoding
+*/
+
+import { 
+  Client, 
+  DaoAction, 
+  RegisterStandardCallbackParams 
+} from "@aragon/sdk-client";
+import { context } from "../index";
+
+// Instantiates a general purpose Client from the Aragon OSx SDK context.
+const client: Client = new Client(context);
+
+const params = {
+  interfaceId: "0x3134r1er213740123741207831238410972347",
+  callbackSelector: "0x382741239807410892375182734892",
+  magicNumber: "0x12192304781237401321329450123321"
+};
+
+const dao: string = "0x123123123123123123123123123123123123";
+
+const action: DaoAction = client.encoding.registerStandardCallbackAction(
+  dao,
+  params
+);
+console.log({ action });
+
+/* MARKDOWN
+
+```json
+  {
```

Comment:
If we log `{ action }`, this is not correct and should be:

```
{ action: {
    to: "0x123123123...",
    value: 0n,
    data: Uint8Array[12,34,45...]
  }
}
```

if we log `action` this is correct